src/init.cpp

```diff
@@ -977,12 +977,12 @@ bool AppInitParameterInteraction()
     // If you are mining, be careful setting this:
     // if you set it to zero then
     // a transaction spammer can cheaply fill blocks using
-    // 1-satoshi-fee transactions. It should be set above the real
+    // 0-fee transactions. It should be set above the real
```

Comment:
Consider also changing the first sentence in this comment: `Fee-per-kilobyte amount considered the same as "free"` doesn't make much sense now that we don't relay free transactions by default.